src/main/java/net/greghaines/jesque/worker/Watchdog.java

```diff
@@ -74,37 +71,52 @@ private void activate() {
 
     private void requeueLostJobs(final String serverName, final String isDebug) {
         // On watchdog activation check whether server contains unhandled in-flight jobs and re-queuing them
-        try {
-            jedis.evalsha(this.watchdogScriptHash.get(), WATCHDOG_SCRIPT_KEYS_NUMBER, serverName, REQUEUE_JOBS, isDebug);
-        } catch (RuntimeException e) {
-            LOG.error("Failed to run " + REQUEUE_JOBS + " job " + WATCHDOG_LUA + " script", e);
-            recoverFromException(e);
-            requeueLostJobs(serverName, isDebug);
+
+        int requeueJobsRetryNum = 0;
+
+
+        while (requeueJobsRetryNum <= REQUEUE_JOBS_RETRY_NUM) {
```

Comment:
Why do we need the retry here?
Let's define the pool that will check the connection before borrow. It's less critical here as this code is running only when a server is going down.